### PR TITLE
My changes

### PR DESCRIPTION
**Pull request by SAnnamalai-techfinite**

### CODE REVIEW

File: frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py [hunk 1]
```diff
@@ -213,5 +213,11 @@ def format_number(self, number):
...
-
-        return number
+        if (number.startswith("91")):
+            pass
+        else:
+            number = f"91{number}"
+        if len(number)==12:
+            return number
+        else:
+            frappe.throw("Please enter the correct 10 digit number")
```
Comment (shridarpatil) [inline frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py:223]: This validation is specific to Indian numbers. We have people from outside India using this app so the validations shouldn't be specific to any country.

File: frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def send_scheduled_message(self) -> dict:
             fieldname='language_code'
         )
         if language_code:
-            for contact in self._contact_list:
+            for contact in self.contact_list:
```
Comment (shridarpatil) [inline frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py:36]: contact_list has to be added as filed in doctype

File: frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_message/whatsapp_message.json [hunk 3]
```diff
@@ -93,8 +93,7 @@
    "fieldname": "content_type",
    "fieldtype": "Select",
    "label": "Content Type",
-   "options": "\ntext\ndocument\nimage\nvideo\naudio",
-   "reqd": 1
+   "options": "\ntext\ndocument\nimage\nvideo\naudio"
```
Comment (shridarpatil) [inline frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_message/whatsapp_message.json:96]: Options are required. The user has to select one of those values.

File: frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py (hunk 2 above)
Comment (SAnnamalai-techfinite) [reply frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_notification/whatsapp_notification.py:36]: There is no such field named _contact_list

**Pull request closed without merging**